helm/consoleme/values.yaml

```diff
@@ -0,0 +1,375 @@
+# via the CLI, and will not see the roles in ConsoleMe's UI. This is useful to keep the UI uncluttered.
+# Example IAM role tags:
+# consoleme-authorized=user1@example.com:group2@example.com
+# consoleme-owner-dl=appowner@example.com
+# consoleme-authorized-cli-only=group3@example.com
+cloud_credential_authorization_mapping:
+  role_tags:
+    enabled: True
+    required_trust_policy_entity: "arn:aws:iam::<AWS account number>:role/ConsoleMeInstanceProfile"
+    authorized_groups_tags:
+      - consoleme-authorized
+    authorized_groups_cli_only_tags:
+      - consoleme-owner-dl
+      - consoleme-authorized-cli-only
+    dynamic_config:
+      enabled: true
+    internal_plugin:
+      enabled: false
+
+account_ids_to_name:
+  "<First AWS account number>": <First AWS account name>
+  "<Second AWS account number>": <Second AWS account name>
+
+aws:
+  issuer: YourCompany
+  account_number: "<Main AWS account number>"
+  region: us-east-1
+
+cli_auth:
+  certificate_header: certificate_header
+  required_headers:
+    - RequiredMTLSHeader: RequiredMTLSHeaderValue
+
+challenge_url:
+  enabled: true
+  request_ip_must_match_challenge_creation_ip: false
+
+consoleme:
+  # Whether to show terraform exporter tab or not
+  export_to_terraform_enabled: false
+  environment: prod
+  development: false
+  url: http://localhost:8081
+  application_admin: admin@example.com
+  jwt_secret: secret
+  web:
+    replicaCount: 1
+    # Not using the script retrieve_or_decode_configuration
+    command: >
+      "FORCE_COLOR=true yarn --cwd /apps/consoleme/ui/ build:prod;
+      watchmedo auto-restart --recursive --pattern='*.py;*.yaml' --directory='.' python consoleme/__main__.py"
+    containerPort: 8081
+    livenessProbe:
+      path: /
+      port: 8081
+    readinessProbe:
+      path: /
+      port: 8081
+    env:
+      - name: EC2_REGION
+        value: us-east-1
+      - name: CONFIG_LOCATION
+        value: /etc/consoleme/config.yaml
+      - name: SETUPTOOLS_USE_DISTUTILS
+        value: stdlib
+    resources:
+      {}
+      # We usually recommend not to specify default resources and to leave this as a conscious
+      # choice for the user. This also increases chances charts run on environments with little
+      # resources, such as Minikube. If you do want to specify resources, uncomment the following
+      # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+      # limits:
+      #   cpu: 1
+      #   memory: 4096Mi
+      # requests:
+      #   cpu: 100m
+      #   memory: 512Mi
+  celery:
+    replicaCount: 1
+    region: us-east-1
+    # Not using the script retrieve_or_decode_configuration
+    # In case you're deploying DynamoDB in this chart you should add this command at the beginning:
+    #     python scripts/initialize_dynamodb_oss.py;
+    # It'll create the necessary tables
+    command: >
```

Comment:
Is it possible to have separate containers for celery schedulers and workers? There should only be 1 scheduler per deployment, but up to N worker containers. The command listed here for a combined celery scheduler and workers, but when we scale, we don't want to scale the number of schedulers.

The command for scheduler (It needs to point to the same CONFIG_LOCATION, and should talk to the same redis instances as the workers):

`celery -A consoleme.celery_tasks.celery_tasks beat -l DEBUG --pidfile /tmp/celery.pid`

A good production-ready command for the workers is:

`celery -A consoleme.celery_tasks.celery_tasks worker -l DEBUG -E --pidfile /tmp/celery.pid --max-memory-per-child=1000000 --max-tasks-per-child 50 --soft-time-limit 3600 --concurrency=10 -O fair`

Reply:
Sure, it's possible to separate them. I'll change it and test here.

Reply:
I pushed the changes. I created a separated deployment to celery worker, now we have a deployment for celery scheduler and another one to workers.

Reply:
Thanks! Sorry for the delay in review. Q2 retro / Q3 planning time. I expect I will take a look at this early next week

Reply:
Sure no problem, I added you as a maintainer of our forked repository, because I don't see the option described in the page above.